src/grblogtools/api.py

```diff
@@ -90,6 +84,23 @@ def summary(self):
                 Log=lambda df: df.apply(strip_model_and_seed, axis=1),
             )
         )
+
```

Comment:
Setting this `_common` attribute would seem to require that summary() is always called before progress()?

Reply:
In the current `get_dataframe` function, there is no option to return `progress` without calling `summary`. But you are right. I think you already took care of this in this [commit](https://github.com/simonbowly/grblogtools/commit/783eb28033c877d60b82219cf3e7ed3b1e8da2f6).